database/exampleWords.js

```diff
@@ -0,0 +1 @@
+module.exports.words = ["Snackwave copper mug wolf adaptogen, ennui tumeric banjo shaman prism dreamcatcher.  Mlkshk chartreuse photo booth skateboard before they sold out copper mug wolf tumeric.  Fixie chambray cray neutra.  Raclette lomo aesthetic single-origin coffee, williamsburg chambray polaroid succulents flannel vice keffiyeh coloring book try-hard.  Letterpress gentrify tofu, VHS pour-over food truck pabst.","Pickled kogi messenger bag, truffaut poutine ramps butcher microdosing.  Lo-fi man bun banh mi, everyday carry kinfolk vegan umami VHS raclette bitters narwhal DIY enamel pin.  Authentic tilde cloud bread distillery salvia affogato.  Organic keytar fam iceland bitters distillery etsy street art.","Meditation chartreuse asymmetrical tofu umami gochujang brunch fixie affogato letterpress taxidermy.  Neutra stumptown 90's man bun butcher photo booth.  Cliche adaptogen forage portland squid listicle kinfolk hot chicken literally neutra cronut.  Kombucha celiac mustache distillery kickstarter.  Williamsburg jean shorts man braid deep v, vape normcore pop-up mustache 3 wolf moon twee messenger bag scenester godard.  Glossier keytar vice, enamel pin hammock live-edge cloud bread truffaut selfies succulents pitchfork migas.  Master cleanse before they sold out vice try-hard, synth salvia mixtape raclette coloring book.","Poutine sartorial quinoa, flannel hella paleo portland meh ethical authentic.  Pickled asymmetrical before they sold out portland mixtape vaporware hammock kickstarter pitchfork pok pok quinoa flannel.  Meditation drinking vinegar listicle edison bulb deep v offal mlkshk.  PBR&B succulents shoreditch marfa messenger bag franzen.","La croix godard mustache, keytar twee man bun offal williamsburg plaid slow-carb disrupt fixie biodiesel pinterest.  Woke deep v umami, +1 viral 90's hexagon portland chartreuse unicorn tattooed freegan sartorial.  Artisan 90's pork belly chillwave tousled, austin cred bicycle rights tumblr tbh +1 helvetica lomo vexillologist hella.  Freegan literally church-key, before they sold out readymade williamsburg humblebrag +1 pitchfork venmo health goth kombucha semiotics bitters."];
```

Comment:
That's interesting that you stored the words here rather than making calls to get random words in your seed script. I'm guessing that's more efficient and less taxing during the data generation. I'm interested to know why you took this route?

Reply:
I just put the output of a call to an API to get a bunch of random words into a file that I could pull from. You're right that I could have made a call to get these random words before seeding instead... that probably would have been the same amount of time. Making a call to get random words with every database seed would have been taxing.